### PR TITLE
Add connection param to the BaseLoginOptions type

### PR DESCRIPTION
**Pull request by luisrudge**

### CODE REVIEW

File: docs/interfaces/auth0clientoptions.html [hunk 1]
```diff
@@ -1141,9 +1167,28 @@ <h3><span class="tsd-flag ts-flagOptional">Optional</span> prompt</h3>
...
+				<aside class="tsd-sources">
+					<ul>
+						<li>Defined in <a href="https://github.com/auth0/auth0-spa-js/blob/339808f/src/global.ts#L46">src/global.ts:46</a></li>
 					</ul>
 				</aside>
+				<div class="tsd-comment tsd-typography">
+					<div class="lead">
+						<p>The default URL where Auth0 will redirect your browser to with
+							the authentication result. It must be whitelisted in
+							the &quot;Allowed Callback URLs&quot; in your Auth0 Application&#39;s
```
Comment (joshcanhelp) [inline docs/interfaces/auth0clientoptions.html]: Add "field"

> the &quot;Allowed Callback URLs&quot; field in your ...

File: docs/interfaces/auth0clientoptions.html [hunk 2]
```diff
@@ -1141,9 +1167,28 @@ <h3><span class="tsd-flag ts-flagOptional">Optional</span> prompt</h3>
...
+							settings. If not provided here, should be provided in the other
```
Comment (joshcanhelp) [inline docs/interfaces/auth0clientoptions.html]: Add "it"

> If not provided here, it should be provided

File: docs/interfaces/auth0clientoptions.html [hunk 3]
```diff
@@ -1063,14 +1071,32 @@ <h3>client_<wbr>id</h3>
...
+				<div class="tsd-signature tsd-kind-icon">connection<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
+				<aside class="tsd-sources">
+					<p>Inherited from BaseLoginOptions.connection</p>
+					<ul>
+						<li>Defined in <a href="https://github.com/auth0/auth0-spa-js/blob/339808f/src/global.ts#L25">src/global.ts:25</a></li>
+					</ul>
+				</aside>
+				<div class="tsd-comment tsd-typography">
+					<div class="lead">
+						<p>The name of the connection configured to your application.
```
Comment (joshcanhelp) [inline docs/interfaces/auth0clientoptions.html]: "to" -> "for"

> ... configured for your application

File: src/global.ts [hunk 4]
```diff
@@ -17,6 +17,12 @@ interface BaseLoginOptions {
    * The default audience to be used for requesting API access.
    */
   audience?: string;
+  /**
+   * The name of the connection configured to your application.
```
Comment (joshcanhelp) [inline src/global.ts]: "to" -> "for"

> ... configured for your application

Comment (luisrudge) [reply src/global.ts]: fixed

File: docs/interfaces/auth0clientoptions.html (hunk 1 above)
Comment (luisrudge) [reply docs/interfaces/auth0clientoptions.html]: this is auto generated, but I fixed in the right file as well

File: docs/interfaces/auth0clientoptions.html (hunk 2 above)
Comment (joshcanhelp) [reply docs/interfaces/auth0clientoptions.html]: @luisrudge - One left

Comment (luisrudge) [reply docs/interfaces/auth0clientoptions.html]: pushed

**Pull request closed without merging**